cluster/engine.go

```diff
@@ -471,17 +473,17 @@ func (e *Engine) updateSpecs() error {
 }
 
 // RemoveImage deletes an image from the engine.
-func (e *Engine) RemoveImage(image *Image, name string, force bool) ([]*dockerclient.ImageDelete, error) {
-	array, err := e.client.RemoveImage(name, force)
+func (e *Engine) RemoveImage(image *Image, name string, force bool) ([]types.ImageDelete, error) {
```

Comment:
What is the purpose if the `image *Image` parameter? It wasn't used before, and it's not used after.